Packs/CrowdStrikeFalcon/TestPlaybooks/Test_Playbook_-_CrowdStrike_Falcon_-_Retrieve_File.yml

```diff
@@ -0,0 +1,1532 @@
+      '#default#':
+      - "264"
+    separatecontext: false
+    conditions:
+    - label: ' Verified'
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: File
+                accessor: SHA256
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2380,
+          "y": -620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "266":
+    id: "266"
+    taskid: 3cb2a192-d130-4846-8cb6-601391d07d7f
+    type: regular
+    task:
+      id: 3cb2a192-d130-4846-8cb6-601391d07d7f
+      version: -1
+      name: Verify Context Error - File SHA512
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      message:
+        simple: |-
+          The 'File.SHA512' context key was not extracted properly. This may indicate that one or more of the following changes have been made to the 'CrowdStrike Falcon - Retrieve File' playbook:
+          1- The 'cs-falcon-rtr-retrieve-file' automation outputs have been modified and no longer contain the 'File.SHA512'' context key.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2770,
+          "y": -400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "267":
+    id: "267"
+    taskid: 0f8e419e-6691-4201-8854-cdeccfb3d3e2
+    type: condition
+    task:
+      id: 0f8e419e-6691-4201-8854-cdeccfb3d3e2
+      version: -1
+      name: Verify File SHA512
+      description: |
+        Verify that the 'File.Size' context key has been extracted correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      ' Verified':
+      - "282"
+      '#default#':
+      - "266"
+    separatecontext: false
+    conditions:
+    - label: ' Verified'
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: File
+                accessor: SHA512
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2770,
+          "y": -620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "268":
+    id: "268"
+    taskid: e9aef9e2-e238-4900-8ec0-867032d04738
+    type: regular
+    task:
+      id: e9aef9e2-e238-4900-8ec0-867032d04738
+      version: -1
+      name: Verify Context Error - File Size
```

Comment:
It's an error for File Name (task 269) rather than File Size, please fix the name.

Reply:
fixed